components/gc-featured-link/_base.scss

```diff
@@ -17,5 +17,9 @@
 		p, a {
 			color: #fff;
 		}
+
+		html:not(.wb-disable) &[data-bg-color] {
+			background-color: transparent;
```

Comment:
We would need to set a default text color which we do know its going to be rewritten when the plugin are initialized.